test/new-e2e/system-probe/system-probe-test-env.go

```diff
@@ -136,6 +143,10 @@ func getAvailabilityZone(env string, azIndx int) string {
 	return ""
 }
 
+func trueStr(t bool) string {
```

Comment:
Use `strconv.FormatBool` instead.

Reply:
preferably remove this function, and use `strconv.FormatBool` inline where needed